CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+### 1.124 - 2024-12-01
+
+##### Deprecated :hourglass_flowing_sand:
+
+- defaultValue() has been deprecated. All uses have been changed to the nullish coalescing operator (??)
```

Comment:
This can go under the 1.123 release since that will be the next one (and I expect we should be able to get this in by then)

Reply:
@jjspace Is the play to stick with the nullish coalescing operator, or deprecating defaultValue and create DefaultValue? Based on #12238, I understood it is the latter. If that is the case, would it make more sense to first create DefaultValue and then replace defaultValue with DefaultValue in the codebase? I saw Dave mention he may need to redo the majority of the PR, so it may be easier to directly replace defaultValue with DefaultValue. Rather than replace defaultValue with ?? to then replace it with DefaultValue. If I misunderstood anything, could you provide some guidance?